vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewModel.kt

```diff
@@ -138,7 +138,7 @@ class MessageComposerViewModel @AssistedInject constructor(
     }
 
     private fun handleOnTextChanged(action: MessageComposerAction.OnTextChanged) {
-            val needsSendButtonVisibilityUpdate = currentComposerText.isEmpty() != action.text.isEmpty()
+            val needsSendButtonVisibilityUpdate = currentComposerText.isBlank() != action.text.isBlank()
```

Comment:
To match `isNotBlank` invocation [here](https://github.com/vector-im/element-android/blob/cad5e732dfb6af7831017bdfb44792138da0521a/vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewModel.kt#L155)